src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -693,7 +693,7 @@ public long GetNumberOfCounterEntries(DocumentsOperationContext context)
             if (hadModifications == false)
                 return;
 
-            var flags = doc.Flags.Strip(DocumentFlags.FromClusterTransaction);
+            var flags = doc.Flags.Strip(DocumentFlags.FromClusterTransaction | DocumentFlags.Resolved);
```

Comment:
Check what happens with attachments in the same scenario

Reply:
done